apis/v1alpha1/httproute_types.go

```diff
@@ -377,6 +383,35 @@ type HTTPRequestHeaderConfig struct {
 	// TODO
 }
 
+// HTTPRequestMirrorFilter defines configuration for the
+// RequestMirror filter.
+type HTTPRequestMirrorFilter struct {
+	// TargetRef is an object reference to forward matched requests to.
+	// The resource may be "services" (omit or use the empty string for the
+	// group), or an implementation may support other resources (for
+	// example, resource "myroutetargets" in group "networking.acme.io").
+	// Omitting or specifying the empty string for both the resource and
+	// group indicates that the resource is "services".  If the referent
+	// cannot be found, the "InvalidRoutes" status condition on any Gateway
+	// that includes the HTTPRoute will be true.
```

Comment:
This condition makes sense for a ForwardTo but may be a bit extreme for this extended MirrorTo functionality. Maybe there's a way to add a condition to the Route itself for this? Any ideas @jpeach?

Reply:
Yeh, it probably makes sense to expose more detail about routes on the route itself, but it's also problematic because different controllers resolving the same routes can get different results. I don't have a concrete suggestion for this yet.